docs/dev/design_documents/ohai_cookbook_segment.md

```diff
@@ -15,22 +15,22 @@ Ohai plugins in all synchronized cookbooks after cookbook synchronization.
 
     As a Chef User,
     I want my Ohai plugins synchronized with my cookbooks,
-    So that I don't incure more unavoidable round-trips to the Chef Server.
+    So that I don't incur more unavoidable round-trips to the Chef Server.
 
     As a Chef Developer,
     I want Ohai plugins as a first-class object,
     So that I don't have to compile recipes to discover templates that drop plugins.
 
 ## Specification
 
-The "segments" of a cookbook will be extended to include an "ohai" segment.  In this segment there will be plugins which are intended to be copied to the Ohai `plugin_path`.  All files in this segment will be copied, recursively, maintaining directory structure.
+The "segments" of a cookbook will be extended to include an "ohai" segment.  In this segment, there will be plugins, which are intended to be copied to the Ohai `plugin_path`. All files in this segment will be copied, recursively, maintaining directory structure.
 
-In the `Chef::RunContext::CookbookCompiler#compile` method a phase will be added after `compile_libraries` and before `compile_attributes` which will copy the Ohai plugins from the cookbook segment and will load all of the discovered plugins.
+In the `Chef::RunContext::CookbookCompiler#compile` method, a phase will be added after `compile_libraries` and before `compile_attributes`, which will copy the Ohai plugins from the cookbook segment and will load all of the discovered plugins.
 
-The plugins will be copied from `<cookbookname>/ohai` into `/etc/chef/ohai/cookbook-plugins/<cookbookname>` as their top level directory (recursively).  The state of the entire
+The plugins will be copied from `<cookbookname>/ohai` into `/etc/chef/ohai/cookbook-plugins/<cookbookname>` as their top level directory (recursively). The state of the entire
 subdirectory tree under `/etc/chef/ohai/cookbook-plugins` will be managed fully by chef-client so that any files which are not synchronized by chef-client will be removed, so
```

Comment:
In line 31, there are two dependent clauses in a row that begins with "so", which is not correct. Is there a part of a sentence missing here?

Reply:
quite the run on here